distributed/tests/test_nanny.py

```diff
@@ -568,3 +568,19 @@ async def test_restart_memory(c, s, n):
 
     while not s.workers:
         await asyncio.sleep(0.1)
+
+
+@gen_cluster(Worker=Nanny, nthreads=[("", 1)], timeout=10)
```

Comment:
```suggestion
@gen_cluster(Worker=Nanny, nthreads=[("", 1)])
```

Reply:
Sure, you're worried 10s might be too short for CI?

Reply:
I just don't see a reason why this particular test should be treated differently